### PR TITLE
tools, op-guide: update configuration and PD Control

### PR DESCRIPTION
**Pull request by lilin90**
@CaitinChen @disksing PTAL

Via the following PRs: 

- https://github.com/pingcap/docs-cn/pull/781
- https://github.com/pingcap/docs-cn/pull/794
- https://github.com/pingcap/docs-cn/pull/796
- https://github.com/pingcap/docs-cn/pull/802
- https://github.com/pingcap/docs-cn/pull/803

### CODE REVIEW

File: tools/pd-control.md [hunk 1]
```diff
@@ -211,27 +306,33 @@ Usage:
...
 {
   "name": "pd",
   "addr": "http://192.168.199.229:2379",
   "id": 9724873857558226554
 }
->> member delete name pd2               // Delete "pd2"
-Success!
->> member delete id 1319539429105371180 // Delete a node using id
-Success!
+>> member leader resign // Move leader away from current members
```
Comment (disksing) [inline tools/pd-control.md]: members -> member, there can only be 1 leader at the same time.

**Comment by disksing** [comment]
the rest LGTM.

**Comment by lilin90** [comment]
@CaitinChen PTAL

File: tools/pd-control.md [hunk 2]
```diff
@@ -132,16 +145,58 @@ Usage:
...
+- `max-merge-region-size` controls the upper limit on the size of Region Merge (the unit is M). When `regionSize` exceeds the specified value, PD does not merge it with the adjacent Region. Setting it to 0 indicates disabling Region Merge.
+
+    ```bash
+    >> config set max-merge-region-size 16 // Set the upper limit on the size of Region Merge to 16M
+    ```
+
+- `max-merge-region-rows` controls the upper limit on the row count of Region Merge. When `regionRowCount` exceeds the specified value, PD does not merge it with the adjacent Region.
+
+    ```bash
+    >> config set max-merge-region-rows 50000 // Set the the upper limit on rowCount to 50k
```
Comment (CaitinChen) [inline tools/pd-control.md]: I think `5000` is better than `50k` because `1k` indicates `1024`.

File: tools/pd-control.md [hunk 3]
```diff
@@ -244,13 +345,15 @@ Usage:
...
->> operator remove remove-peer 1 2          // Remove a replica of region 1 on store 2
->> operator add transfer-leader 1 2         // Schedule the leader of region 1 to store 2
->> operator add transfer-region 1 2 3 4     // Schedule region 1 to store 2,3,4
->> operator add transfer-peer 1 2 3         // Schedule the replica of region 1 on store 2 to store 3
->> operator remove 1                        // Remove the scheduling operation of region 1
+>> operator show region                     // Display all Region operators
+>> operator add add-peer 1 2                // Add a replica of Region 1 on store 2
+>> operator remove remove-peer 1 2          // Remove a replica of Region 1 on store 2
+>> operator add transfer-leader 1 2         // Schedule the leader of Region 1 to store 2
+>> operator add transfer-region 1 2 3 4     // Schedule Region 1 to store 2,3,4
```
Comment (CaitinChen) [inline tools/pd-control.md]: store 2,3,4 -> **stores** 2,3,4

**Pull request merged**